test/plugin_fixtures/unique_string_aggregator_plugin.yml

```diff
@@ -0,0 +1,73 @@
+unique_string_aggregator_plugin:
+  api_namespace: movies
```

Comment:
@alis-khadka we should audit the names of these api_namespaces and provide a naming scheme to avoid collision and keep everything easy to understand